turbopack/crates/turbo-persistence/src/compaction/selector.rs

```diff
@@ -1,17 +1,6 @@
-/// The merge and move jobs that the compaction algorithm has computed. It's expected that all move
-/// jobs are executed in parallel and when that has finished the move jobs are executed in parallel.
-#[derive(Debug)]
-pub struct CompactionJobs {
-    pub merge_jobs: Vec<Vec<usize>>,
-    pub move_jobs: Vec<usize>,
-}
+use smallvec::{SmallVec, smallvec};
 
-impl CompactionJobs {
-    #[cfg(test)]
-    pub(self) fn is_empty(&self) -> bool {
-        self.merge_jobs.is_empty() && self.move_jobs.is_empty()
-    }
-}
+use crate::compaction::interval_map::IntervalMap;
 
 type Range = (u64, u64);
```

Comment:
Could this just be an `std::ops::Range`? https://doc.rust-lang.org/std/ops/struct.Range.html

Reply:
RangeInclusive maybe, but it has a weird internal representations. A tuple works for us.